faq.html

```diff
@@ -87,15 +87,15 @@ <h4 class="panel-title">
               <div class="panel-body">
                 <p>Open the drawer by swiping from the left or tapping the <i class="fa fa-bars"></i> burger icon on the top left corner. Go to <i class="fa fa-plus"></i> <b>Add Podcast</b>.<br />Now, you can either 
                   search the iTunes catalog, browser gpodder.net or add a podcast by URL.</p>
-                  <p>You can also click RSS links (and other protocols) from the browser and select AntennaPod to open them.</p>
+                  <p>You can also import feeds by opening protocols such as RSS with AntennaPod.</p>
```

Comment:
"You can also import RSS and Atom feeds as well as URL schemes like `<tt>pcast://</tt>` and `<tt>itpc://</tt>` with AntennaPod"